cate/core/opimpl.py

```diff
@@ -704,13 +704,13 @@ def subset_spatial_impl(ds: xr.Dataset,
 
     lon_min, lat_min, lon_max, lat_max = extents
 
-    # Validate the bounding box
     if (not (-90 <= lat_min <= 90)) or \
             (not (-90 <= lat_max <= 90)) or \
             (not (-180 <= lon_min <= 180)) or \
             (not (-180 <= lon_max <= 180)):
         raise ValueError('Provided polygon extends outside of geospatial'
-                         ' bounds: latitude [-90;90], longitude [-180;180]')
+                         '\nbounds : latitude [-90;90], longitude [-180;180]'
+                         f'\nrequest: latitude [{lat_min};{lat_max}], longitude[{lon_min};{lon_max}]')
```

Comment:
The new line symbols here are redundant in my opinion. The string is split simply to make the code line shorter than 120 symbols. That's a code convention we have adopted. Otherwise I don't think we should add new lines, but always print error messages as a single line that can then be wrapped by whatever displays it.

Reply:
@papesci please remove newline characters from error message